policy.md

```diff
@@ -0,0 +1,47 @@
+# Policy
+
+TBD: Things to cover in the introduction:
+
+  - Commercial use and re-use (wasn't in original outline, but should
+    be represented in this section).
+
+  - Importance of distinguishing between trademarks, copyrights, and
+    patents.
+
+## Copyright
+
+TBD
+
+## Trademark
+
+TBD
+
+## Patent
+
+TBD
+
+## Commercial Use
+
+TBD
+
+## Security
+
+TBD: Security policy comes from threat modeling, risk assessment, and
+cost/benefit analysis.  In Security, distinguish what's discussed in
+this section from what's discussed in the "Security" subsection in
+Adoptability.
+
```

Comment:
I understand here we will include recommendations with examples on the policies that need to be enforced to maximize security

Reply:
*nod*  That helps, thank you.  As per [commit 36e40e95](https://github.com/unicef/publicgoods-toolkit/pull/24/commits/36e40e9544f361967b7ff8c5f655aa19111b0472), moved above comment to become an inline TBD now.